kubeadm/buildx.psm1

```diff
@@ -0,0 +1,60 @@
+function Set-Builder()
+{
+    $env:DOCKER_CLI_EXPERIMENTAL = "enabled"
+    & docker buildx create --name img-builder --use --driver docker-container --driver-opt image=moby/buildkit:v0.7.2
+}
+
+function New-Build()
+{
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$name,
+        [Parameter(Mandatory = $true)]
+        [ValidateSet("docker", "registry")]
+        [string]$output,
+        [Parameter(Mandatory = $true)]
+        [string[]]$args
+    )
+
+    $command = "docker buildx build --platform windows/amd64 --output=type=$output -f Dockerfile -t $name"
+    foreach($arg in $args)
+    {
+        $command = "$command --build-arg=$arg"
+    }
+    $command = "$command ."
+    Write-Host $command
+    Invoke-Expression $command
+}
+
+function Push-Manifest([string]$name, [string[]]$items, [string[]]$bases)
+{
+    $command = "docker manifest create $name";
+    foreach($item in $items)
+    {
+        $command = "$command --amend $item"
+    }
+    Write-Host $command
+    Invoke-Expression $command
+
+    for ($i = 0; $i -lt $items.Length; $i++) {
```

Comment:
Can we just add a note this is work around until the next major release for docker cli in 20.x : https://github.com/docker/cli/pull/2578

Reply:
Yes. I did it.